[1.7] FIXME: remove the s/:/./ code

--- origsrc/vnc_unixsrc/tightvncserver.conf	2009-02-11 22:27:18.000000000 -0600
+++ src/vnc_unixsrc/tightvncserver.conf	2009-03-06 03:26:54.337500000 -0600
@@ -19,16 +19,16 @@
 # $geometry = "1024x768";
 # $depth = 24;
 # $desktopName = "X";
-# $vncClasses = "/usr/local/vnc/classes";
+# $vncClasses = "/usr/share/tightvnc/classes";
 # $vncUserDir = "$ENV{HOME}/.vnc";
-# $fontPath = "unix/:7100";
+# $fontPath = "unix/:-1";
 # $authType = "-rfbauth $vncUserDir/passwd";
-# $colorPath = "/usr/lib/X11/rgb";
+# $colorPath = "/usr/share/X11/rgb.txt";
 
 ## Here is another example of setting the font path:
 #
-# $fontPath = "/usr/lib/X11/fonts/misc/"
-# $fontPath = "$fontPath,/usr/lib/X11/fonts/75dpi/";
+# $fontPath = "/usr/share/fonts/misc/"
+# $fontPath = "$fontPath,/usr/share/fonts/75dpi/";
 
 ## You might wish to make your vnc directory under /tmp, to make sure
 ## passwords are always kept on the local filesystem. To do that, just
--- origsrc/vnc_unixsrc/vncserver	2009-02-11 22:27:18.000000000 -0600
+++ src/vnc_unixsrc/vncserver	2009-03-06 03:29:48.150000000 -0600
@@ -32,10 +32,13 @@
 $geometry = "1024x768";
 $depth = 24;
 $desktopName = "X";
-$vncClasses = "/usr/local/vnc/classes";
+$vncClasses = "/usr/share/tightvnc/classes";
 $vncUserDir = "$ENV{HOME}/.vnc";
-$fontPath = "unix/:7100";
+$fontPath = "unix/:-1";
 $authType = "-rfbauth $vncUserDir/passwd";
+$colorPath = "/usr/share/X11/rgb.txt";
+$fontPath = "/usr/share/fonts/misc/,/usr/share/fonts/75dpi/";
+
 
 # Read configuration from the system-wide and user files if present.
 
@@ -141,7 +144,7 @@ if ((@ARGV > 0) && ($ARGV[0] =~ /^:(\d+)
 
 $vncPort = 5900 + $displayNumber;
 
-$desktopLog = "$vncUserDir/$host:$displayNumber.log";
+$desktopLog = "$vncUserDir/$host.$displayNumber.log";
 unlink($desktopLog);
 
 # Make an X server cookie - use as the seed the sum of the current time, our
@@ -179,7 +182,7 @@ $cmd .= " >> " . &quotedString($desktopL
 
 # Run $cmd and record the process ID.
 
-$pidFile = "$vncUserDir/$host:$displayNumber.pid";
+$pidFile = "$vncUserDir/$host.$displayNumber.pid";
 system("$cmd & echo \$! >$pidFile");
 
 # Give Xvnc a chance to start up
@@ -461,11 +464,12 @@ sub Usage
 sub Kill
 {
     $opt{'-kill'} =~ s/(:\d+)\.\d+$/$1/; # e.g. turn :1.0 into :1
+    $opt{'-kill'} =~ s/:/./; # ':' is an illegal filename char on some systems
 
-    if ($opt{'-kill'} =~ /^:\d+$/) {
+    if ($opt{'-kill'} =~ /^\.\d+$/) {
 	$pidFile = "$vncUserDir/$host$opt{'-kill'}.pid";
     } else {
-	if ($opt{'-kill'} !~ /^$host:/) {
+	if ($opt{'-kill'} !~ /^$host\./) {
 	    die "\nCan't tell if $opt{'-kill'} is on $host\n".
 		"Use -kill :<number> instead\n\n";
 	}
